openreview/conference/builder.py

```diff
@@ -1533,6 +1536,62 @@ def is_release_authors(is_note_accepted):
             self.set_homepage_decisions(decision_heading_map=decision_heading_map)
         self.client.remove_members_from_group('active_venues', self.id)
 
+    def post_decisions(self, decisions_file):
+        decisions_data = list(csv.reader(StringIO(decisions_file.decode()), delimiter=","))
+        decision_notes = {
+            n.forum: n for n in self.client.get_all_notes(
+                invitation=self.get_invitation_id(self.decision_stage.name, '.*')
+            )}
+
+        paper_notes = {
+            n.forum: n for n in self.client.get_all_notes(
+                invitation=self.get_invitation_id('Blind_Submission')
+            )}
+
+        def post_decision(paper_decision):
+            if len(paper_decision) == 3:
+                paper_id, decision, comment = paper_decision
+            elif len(paper_decision) == 2:
+                paper_id, decision = paper_decision
+                comment = ''
+            else:
+                raise ValueError(
+                    "Too many values provided in the decision file. Expected values are: paper_id, decision, comment"
+                )
+
+            paper_note = paper_notes.get(paper_id, None)
+            if not paper_note:
+                raise OpenReviewException(
+                    f"Paper with ID: {paper_id} not found. Please check the submitted paperIDs."
+                )
+
+            paper_decision_note = decision_notes.get(paper_id, None)
+            if paper_decision_note:
+                paper_decision_note.content = {
+                    'title': 'Paper Decision',
+                    'decision': decision.strip(),
+                    'comment': comment,
+                }
+            else:
+                paper_decision_note = openreview.Note(
+                    invitation='{}/Paper{}/-/Decision'.format(self.id, paper_note.number),
```

Comment:
please get the "Decision" world from `decisiont_stage.name` and the value "Paper" is parametrized so you can use the following:

```
self.get_invitation_id(name=decision_stage.name, number=paper_note.number)
```